packages/lib-bitcoinjs/index.ts

```diff
@@ -52,9 +77,17 @@ export class DfnsWallet implements SignerAsync {
   }
 
   public async sign(hash: Buffer): Promise<Buffer> {
+    return this._sign(hash)
+  }
+
+  public async signSchnorr(hash: Buffer): Promise<Buffer> {
+    return this._sign(hash, Buffer.alloc(0))
+  }
+
+  private async _sign(hash: Buffer, taprootMerkleRoot?: Buffer): Promise<Buffer> {
     const res = await this.dfnsClient.wallets.generateSignature({
       walletId: this.metadata.id,
-      body: { kind: 'Hash', hash: `0x${hash.toString('hex')}` },
+      body: { kind: 'Hash', hash: `0x${hash.toString('hex')}`, taprootMerkleRoot: taprootMerkleRoot && taprootMerkleRoot.toString('hex') },
```

Comment:
nit: add a `0x` prefix to the merkle root hex. our api is lenient on the hex formatting, but our own lib should try to be formatting consistent.

```suggestion
      body: { kind: 'Hash', hash: `0x${hash.toString('hex')}`, taprootMerkleRoot: taprootMerkleRoot && `0x${taprootMerkleRoot.toString('hex')}` },
```